ansible/deploy.yml

```diff
@@ -133,20 +153,26 @@
         msg: Masternode not defined in network config
       when: masternode is not defined
   roles:
-    - role: dashd
-      dashd_listen: true
-      dashd_zmq: true
-      dashd_indexes: true
-      tags:
-        - dashd
-    - mn-sentinel
-    - role: insight
-      ui: false
-      when: evo_services
-    - role: mn-evo-services
-      tags:
-        - evo
-      when: evo_services
```

Comment:
We still need this stuff

Reply:
Yeah, committed by accident after trying to run it